storage/src/vespa/storage/storageserver/servicelayernode.cpp

```diff
@@ -105,20 +105,9 @@ ServiceLayerNode::initializeNodeSpecific()
     // node state.
     NodeStateUpdater::Lock::SP lock(_component->getStateUpdater().grabStateChangeLock());
     lib::NodeState ns(*_component->getStateUpdater().getReportedNodeState());
-    ns.setDiskCount(1u);
 
     ns.setCapacity(_serverConfig->nodeCapacity);
     ns.setReliability(_serverConfig->nodeReliability);
-    for (uint16_t i=0; i<_serverConfig->diskCapacity.size(); ++i) {
-        if (i >= ns.getDiskCount()) {
-            LOG(warning, "Capacity configured for partition %zu but only %u partitions found.",
-                _serverConfig->diskCapacity.size(), ns.getDiskCount());
-            continue;
-        }
-        lib::DiskState ds(ns.getDiskState(i));
-        ds.setCapacity(_serverConfig->diskCapacity[i]);
-        ns.setDiskState(i, ds);
-    }
     LOG(debug, "Adjusting reported node state to include partition count and states, capacity and reliability: %s",
```

Comment:
Consider updating log message to no longer mention partition

Reply:
Fixed.